UM/Qt/qml/UM/Slider.qml

```diff
@@ -0,0 +1,132 @@
+import QtQuick 2.7
```

Comment:
I don't mind this, but previously these kind of components did not live in uranium. I'm fine that this lives here, I think it is logical. But I think this should be a consciousness decision and that the rest of the team should be aware.